internal/app/mapper/mapper.go

```diff
@@ -35,7 +35,9 @@ type mapper struct {
 }
 
 const (
-	separator = "_"
+	edsTypeURLV2 = "type.googleapis.com/envoy.api.v2.ClusterLoadAssignmentr"
```

Comment:
one last nit: could we use https://github.com/envoyproxy/go-control-plane/blob/master/pkg/resource/v2/resource.go#L16 and https://github.com/envoyproxy/go-control-plane/blob/master/pkg/resource/v3/resource.go#L14 instead of creating types here?